### DISCUSSION TITLE
CSS and unused CSS

### DISCUSSION THREAD

**Discussion by aleksaar** [GoogleChrome/lighthouse#12743] [Q&A] [answered]
Why is the unused CSS larger than the actual CSS loaded by some pages? For example youtube: CSS loaded 51 kB, unused CSS 273 kB.

**Comment by patrickhulce** [comment, marked as answer]
Can you share an example report that has this issue? That sounds like a bug (unless you're not counting CSS shipped in other resources in the 51 kB which the unused CSS does, for example styles that are contained in strings in JavaScript or inlined into other HTML).

**Reply by aleksaar** [reply]
With the CSS loaded i meant the resource summary audit in CLI, but I did not realise that does not include CSS loaded in other resources.
